apps/gitness/src/pages-v2/pull-request/pull-request-conversation.tsx

```diff
@@ -440,10 +440,14 @@ export default function PullRequestConversationPage() {
       head_branch: pullReqMetadata?.source_branch,
       head_commit_sha: pullReqMetadata?.source_sha
     }
-    rebaseBranch({ body: payload, repo_ref: repoRef }).then(() => {
-      onPRStateChanged()
-      setRuleViolationArr(undefined)
-    })
+    rebaseBranch({ body: payload, repo_ref: repoRef })
+      .then(() => {
+        onPRStateChanged()
+        setRuleViolationArr(undefined)
+      })
+      .catch(error => {
+        alert(error.message)
```

Comment:
we should remove `alerts` and console.logs. 
either error should be shown to the user, or sent to our BE (we don't have that support yet in gitness though)